float32 cur_vel
float32 tar_vel
float32 tar_dist
bool alpha
bool send_rear_camera_image
uint8 lrc_mode
uint8 crc_mode
